jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -223,6 +225,11 @@ def _on_is_active_changed(self, *args):
     def _on_viewer_key_event(self, viewer, data):
         if data['event'] == 'keydown' and data['key'] == 'm':
             row_info = self.coords_info.as_dict()
+            if 'value' in row_info:
+                # format and cast flux value for Table
+                row_info['value'] = f"{row_info['value']:.5e}"
+            else:
+                row_info['value'] = ''

```

Comment:
why is this necessary?  Can/should we move it to `as_dict` instead?

Reply:
There was an issue arising where the value column in the table couldn't be ready properly. This was happening when casting value when it was numpy.float32 to float in coords_info.py. I added a check to only cast if the value is not already float.